data.py

```diff
@@ -166,6 +202,8 @@ def get_data(
         evaluation_data_points = prepare_human_eval()
     elif dataset == DatasetFormat.CUSTOM_JSONL:
         evaluation_data_points = prepare_custom(data_path, prompt_field=prompt_field, response_field=response_field)
+    elif dataset == DatasetFormat.TOP_v2:
```

Comment:
I suggest to use all uppercase to be consistent with the used enum convention

```suggestion
    elif dataset == DatasetFormat.TOP_V2:
```